lessons/pydata/pandas_types/index.ipynb

```diff
@@ -0,0 +1,10723 @@
+{
```

Comment:
<p>Asi bych lidi nezatěžoval binárníma formátama. Myslim že pro začátečníky je s nima víc problémů než užitku. A pro jejich datasety asi problém s výkonem nebude. CSV je v pohodě, navíc to bude jednoduše shareovatelné.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='127'/>

Reply:
<p>"Excel ani CSV nejsou formáty pro ukládání velikých dat zcela vhodné (jako alternativy se nabízí třeba&nbsp;<a href="https://github.com/wesm/feather" target="_blank">feather</a>&nbsp;nebo&nbsp;<a href="https://en.wikipedia.org/wiki/Apache_Parquet" target="_blank">parquet</a>), pro naše účely (malé soubory, čitelný textový formát) ale budou CSV postačovat."</p>

Reply:
jop, super